components/elm/cime_config/testdefs/testmods_dirs/elm/ctc_f19_g16_I1850GSWCNPRDCTCBC/user_nl_elm

```diff
@@ -1,2 +1,2 @@
-finidat             = '$DIN_LOC_ROOT/lnd/clm2/initdata_map/clmi.I1850GSWCNPRDCTCBC.f19_g16.67a0f98.0401-01-01-00000.nc'
+finidat             = '$DIN_LOC_ROOT/lnd/clm2/initdata_map/clmi.I1850GSWCNPRDCTCBC.f19_g16.67a0f98.noNaN.0401-01-01-00000.nc'
```

Comment:
How are these `finidat` files generated?

Reply:
There was only one variable that needed modification which was `btran2` so I edited the previous finidat file to replace all `nan` with the fill value instead using python netCDF4